BigQuery/tests/System/ManageJobsTest.php

```diff
@@ -47,6 +47,33 @@ public function testListJobs()
         return $job;
     }
 
+    public function testListJobsWithTimeFilter()
+    {
+        $query = self::$client->query(sprintf(
+            'SELECT * FROM [%s.%s]',
+            self::$dataset->id(),
+            self::$table->id()
+        ));
+        $job = self::$client->startQuery($query);
+        $info = $job->info();
+        $jobId = $job->id();
+        $creationTime = $info['statistics']['creationTime'];
+        $jobs = self::$client->jobs([
+            'maxCreationTime' => $creationTime + 1000,
```

Comment:
Oh sorry, just found out this test might be flaky because the filter may hit the other job created by other test.